assets/app/views/builds.html

```diff
@@ -114,39 +115,39 @@
                       </dd>
                   </div>
                   <div ng-switch-when="imageChange">
-                    <div ng-switch="buildConfig.parameters.strategy.type">
-                      <div ng-switch-when="STI">
-                        <dl class="dl-horizontal" ng-if="buildConfig.parameters.strategy.stiStrategy.from&& buildConfig.parameters.strategy.stiStrategy.from.kind!='ImageStreamImage'">
+                    <div ng-switch="buildConfig.spec.strategy.type">
+                      <div ng-switch-when="Source">
+                        <dl class="dl-horizontal" ng-if="buildConfig.spec.strategy.sourceStrategy.from && buildConfig.spec.strategy.sourceStrategy.from.kind!='ImageStreamImage'">
                           <dt>
                             New image for:
                           </dt>
                           <dd>
-                            Image stream {{buildConfig.parameters.strategy.stiStrategy.from | imageRepoReference : buildConfig.parameters.strategy.stiStrategy.from.kind : buildConfig.parameters.strategy.stiStrategy.Tag}}
+                            Image stream {{buildConfig.spec.strategy.sourceStrategy.from | imageRepoReference : buildConfig.spec.strategy.sourceStrategy.from.kind : buildConfig.spec.strategy.sourceStrategy.from.name}}
```

Comment:
In v1beta3, there's no separate tag field, if it's there, it will be part of the name when kind is ImageStreamTag. That last param for the filter should be an empty string ""